pxl_scripts/sotw/dns_external_fqdn_list/data.pxl

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' DNS Traffic FQDN List
+Lists external, fully qualified domain names (FQDNs) from all DNS traffic on the cluster for a specified amount of time.
+'''
+
+import px
+
+def fqdn_list(start: str):
+
+    # The dns_events table pairs DNS requests with their responses.
+    df = px.DataFrame(table='dns_events', start_time=start)
+
+    # Parse the DNS response to determine if it was successfully resolved.
+    df.resp_body = px.pluck(df.resp_body, 'answers')
+    df.request_resolved = px.contains(df.resp_body, 'name')
+
+    # Filter for resolved requests.
+    df = df[df.request_resolved == True]
+
+    # Parse the DNS request for query name.
+    # TODO: cleanup this logic when we support object types.
+    df.req_body = px.pluck(df.req_body, 'queries')
+    df.idx1 = px.find(df.req_body, '\"name\":')
+    df.fqdn_partial = px.substring(df.req_body, df.idx1 + 8, px.length(df.req_body) - (df.idx1 + 8) - 3)
+    df.idx2 = px.find(df.fqdn_partial, ',')
+    df.fqdn = px.substring(df.fqdn_partial, 0, df.idx2 - 1)
+
+    # Filter out local domains quieres by excluding fqdns with known local domain suffix.
```

Comment:
How about an additional comment like "If your k8s cluster uses a different internal domain suffix, add it here."